src/app/core/components/archive-payer/archive-payer.component.html

```diff
@@ -0,0 +1,40 @@
+<!-- @format -->
+<div class="archive-payer">
+  <p>
+    When an Owner or Manager becomes the payer of an archive, any subsequent
+    uploads will count against their personal storage balance.
+  </p>
+  <div *ngIf="!hasPayer">
+    <p>This archive does not currently have an assigned payer.</p>
+  </div>
+  <div *ngIf="hasPayer">
+    <p *ngIf="!isPayerDifferentThanLoggedUser">You are this archive's payer.</p>
+    <div *ngIf="isPayerDifferentThanLoggedUser">
+      <p>This archive's payer is:</p>
+      <div class="payer-info">
+        <p class="payer-name">
+          <b>{{ payer.fullName }}</b>
+        </p>
+        <p class="payer-email">{{ payer.primaryEmail }}</p>
+        <p class="payer-access">Access: {{ roles[payer.accessRole] }}</p>
+      </div>
+      <p>Contact this member for archive storage requests.</p>
+    </div>
+  </div>
+  <div class="toggler">
+    <pr-switcher
+      [data]="{
+        archiveId: this.archive?.archiveId,
+        isPayerDifferentThanLoggedUser: this.isPayerDifferentThanLoggedUser,
+        handleAccountInfoChange: this.handleAccountInfoChange.bind(this),
+        cancelAccountPayerSet: this.cancelAccountPayerSet.bind(this)
+      }"
+      [isChecked]="!isPayerDifferentThanLoggedUser"
+      [change]="setArchivePayer"
+    ></pr-switcher>
+    <div>
+      <p class="assign-user">Assign myself as the payer</p>
```

Comment:
This almost looks like a link to me -- is it intended to be clickable in the design, @k8lyn6? (I thought maybe I'd be able to click on it to assign myself as a payer.)

Reply:
More relevant for code review, @crisnicandrei: this says "Assign myself as the payer" even if the logged in user is already the payer. That's a small bug that is present in the design, too -- I'm not sure how it should look/what it should say, but I think it's confusing as is. What do you think, @k8lyn6 ?

Reply:
maybe remove the text if you are the payer? Or if you are the payer something like "Unassign yourself"?

Reply:
<img width="540" alt="Screenshot 2023-06-28 at 13 37 25" src="https://github.com/PermanentOrg/web-app/assets/62384997/24dc57b8-ec12-4845-b1dc-7f5739b85d4f">

Reply:
I think you're right about both, @cecilia-donnelly.  I talked to Nathalie R. and I think the changes we should make here are:

- the "Assign myself as the payer" should be gray when the toggle is off, and blue when the toggle is on
- the "Assign myself as the payer" is clickable, and clicking it turns on or off the toggle
- change "Assign my as the payer" to "Remove myself as the payer" when the toggle is on

I think Nathalie made all of the above changes in the design if that description is confusing, too.

I haven't QA'ed this yet since it looks like there are some code review changes first. Let me know when those changes are made and I'll QA it!

Reply:
@k8lyn6 shouldn’t the text be blue at all times? For me a gray text might suggest that the switch is disabled. Let me know what you think about this!

Reply:
@crisnicandrei ooh good point. Let's just keep it blue the whole time and clickable then. Sorry for designing on the fly!